controllers/drplacementcontrol.go

```diff
@@ -1337,7 +1337,7 @@ func (d *DRPCInstance) checkPVsHaveBeenRestored(homeCluster string) (bool, error
 	vrg, err := d.reconciler.MCVGetter.GetVRGFromManagedCluster(d.instance.Name,
 		d.instance.Namespace, homeCluster, annotations)
 	if err != nil {
-		return false, fmt.Errorf("failed to VRG using MCV (error: %w)", err)
+		return false, fmt.Errorf("waiting for PVs to be restored to failover cluster (status: %w)", err)
```

Comment:
Two things:
1. This function is called from places, the failover and relocate.
2. This is an error retrieving the VRG from the ManagedCluster. Therefore, the word `status` maybe changed to `error`.
It is a good change, but I would rephrase it as:
```
fmt.Errorf("waiting for PVs to be restored on cluster %s (error: %w)", homeCluster, err)
```

Reply:
I refrained from `error` as it appears in the condition message, hence decided `status` is a little less alarming, and still carries the reason (failed to find VRG or the required status from MCV). Hope that works.

Reply:
Okay, I am good with (2) above to stay as is.

Reply:
Ah yes, I see it now. The new message is better, adding the cluster name is more informational as such in this situation. @jmishrarh as this is merged, can you open a new PR with the amended content, thanks!